packages/extension-chakra-storefront/src/components/with-layout/with-layout.tsx

```diff
@@ -292,8 +298,8 @@ const withLayout = <P extends object>(WrappedComponent: React.ComponentType<P>)
                                     {/* TODO: mobile menu */}
                                     <HideOnDesktop>
                                         <DrawerMenu
-                                            isOpen={open}
-                                            onClose={onClose}
+                                            isOpen={isDrawerMenuOpen}
+                                            onClose={onDrawerMenuClose}
```

Comment:
open and onClose were set above so I think this part can remain unchanged?

Reply:
I changed the name to make it clearer since we have a lot open/close state on the _app config.